### PR TITLE
Create tier account and tier config functionality.

### PR DESCRIPTION
**Pull request by martinconstante**

### CODE REVIEW

File: src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    @DisplayName("Update Tier Config Request")
+    public CBCTierConfigRequest updateTierConfigRequest(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Update Tier Account Request")
+        UpdateTierConfigRequestParameter newRequestParameter
+    ) throws CBCException {
+        return (CBCTierConfigRequest) connection.newQ(new TypeReference<CBCTierConfigRequest>() {})
+            .collection("tier")
+            .collection("config-requests",newRequestParameter.getTierConfigRequestId())
+            .action("", HttpMethod.PUT, newRequestParameter.getParams());
```
Comment (rahulmondal) [inline src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java:202]: .update(newRequestParameter.getParams())

File: src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    @DisplayName("Accept Tier Account Request")
+    public CBCAccountRequest acceptTierAccountRequest(
+            @Connection CBCConnection connection,
+            @ParameterGroup(name="Accept Tier Account Request")
+                    AcceptTierAccountRequestParameter newRequestParameter
+    ) throws CBCException {
+        return (CBCAccountRequest) connection.newQ(new TypeReference<CBCAccountRequest>() {})
+            .collection("tier")
+            .collection("account-requests",newRequestParameter.getTierAccountRequestId())
+            .action("accept", HttpMethod.POST,"");
```
Comment (rahulmondal) [inline src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java:87]: .action("accept", HttpMethod.POST, null);

File: src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    @DisplayName("Inquire Tier Config Request")
+    public CBCTierConfigRequest inquireTierConfigRequest(
+        @Connection CBCConnection connection,
+        @ParameterGroup(name="Inquire Tier Account Request")
+        ActionTierConfigRequestParameter newRequestParameter
+    ) throws CBCException {
+        return (CBCTierConfigRequest) connection.newQ(new TypeReference<CBCTierConfigRequest>() {})
+            .collection("tier")
+            .collection("config-requests",newRequestParameter.getTierConfigRequestId())
+            .action("inquire", HttpMethod.POST,"");
```
Comment (rahulmondal) [inline src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java:163]: .action("inquire", HttpMethod.POST, null);

And similar in below code as well.

**Review by rahulmondal — CHANGES_REQUESTED** [review]

File: src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java (hunk 1 above)
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java:202]: Done

File: src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java (hunk 3 above)
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java:163]: Done

File: src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java (hunk 2 above)
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/internal/operations/TierOperations.java:87]: Done

**Review by rahulmondal — CHANGES_REQUESTED** [review]

File: src/main/java/com/cloudblue/connect/api/parameters/accounts/AcceptTierAccountRequestParameter.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package com.cloudblue.connect.api.parameters.accounts;
```
[the remark this answers is not in the document]
Comment (martinconstante) [reply src/main/java/com/cloudblue/connect/api/parameters/accounts/AcceptTierAccountRequestParameter.java]: Done

**Review by rahulmondal — APPROVED** [review]

**Pull request merged**